src/quantum/azext_quantum/commands.py

```diff
@@ -94,6 +98,30 @@ def one(key, value):
         histogram = results['histogram']
         return [one(key, histogram[key]) for key in histogram]
 
+    elif 'errorData' in results:
+        notFound = 'Not found'
+        status = notFound
+        errorCode = notFound
+        errorMessage = notFound
+        target = notFound
+        jobId = notFound
+        submissionTime = notFound
+        if 'status' in results:
+            status = results['status']
+        errorData = results['errorData']
+        if 'code' in errorData:
+            errorCode = errorData['code']
+        if 'message' in errorData:
+            errorMessage = errorData['message']
+        if 'target' in results:
+            target = results['target']
+        if 'id' in results:
+            jobId = results['id']
+        if 'creationTime' in results:
+            submissionTime = results['creationTime']
```

Comment:
```suggestion
    elif 'errorData' in results:
        notFound = 'Not found'
        errorData = results['errorData']
        status = results.get('status', notFound)
        errorCode = errorData.get('code', notFound)
        errorMessage = errorData.get('message', notFound)
        target = results.get('target', notFound)
        jobId = results.get('id', notFound)
        submissionTime = results.get('creationTime', notFound)
```

Reply:
Good call Zelin -- My original 18 lines are condensed into 6, and it still protects against raising a KeyError like my more verbose series of “if…in” statements would.  Thanks! 👍